tests/Example/ReadmeExampleTest.php

```diff
@@ -11,24 +11,24 @@
 
 namespace GraphAware\Neo4j\Client\Tests\Example;
 
-class ReadmeExamleTest extends ExampleTestCase
+class ReadmeExampleTest extends ExampleTestCase
```

Comment:
again, php-cs-fixer's fix, and it does fix a typo : the same of the class was wrong compared to the file name :)